src/core/src/manifest.rs

```diff
@@ -26,7 +26,7 @@ pub struct Record {
 
     md5short: String,
 
-    #[getset(get = "pub", set = "pub")]
+    #[getset(get_copy = "pub", set = "pub")]
```

Comment:
Change this to `get_copy`, for basic types like `u32` and `bool` it makes the API easier (they are trivial to copy)